### PR TITLE
MGMT-3502 Add linter for swagger

### PR DESCRIPTION
**Pull request by avishayt**
Main swagger changes:
- Quotes around return codes
- Descriptions for all parameters
- Declare tags globally
- Add license
- Fixed mistake in upload logs parameters

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -3,11 +3,28 @@ info:
   description: 'Assisted installation'
   version: 1.0.0
   title: AssistedInstall
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
 host: api.openshift.com
 basePath: /api/assisted-install/v1
 tags:
   - name: Assisted installation
```
Comment (YuviGold) [inline swagger.yaml:12]: Let's standardize the tags' names. camelCase/snake_case..

File: .spectral.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+extends: [[spectral:oas, all]]
+rules:
```
Comment (YuviGold) [inline .spectral.yml:2]: Very impressive tool.
After looking into the [OpenAPI rules](https://meta.stoplight.io/docs/spectral/docs/reference/openapi-rules.md) I think we should add also:
* operation-operationId-unique
* operation-2xx-response
* path-params
* no-$ref-siblings
* openapi-tags
* path-keys-no-trailing-slash

Comment (avishayt) [reply .spectral.yml:2]: Agree, I wanted to start this commit with a minimum and add more later

File: swagger.yaml (hunk 1 above)
Comment (avishayt) [reply swagger.yaml:12]: Yes I agree, just didn't want to make "real" changes in this commit

**Pull request merged**